### ISSUE TITLE
Difference in output directory structure between cbuild and csolution convert

### ISSUE THREAD

**Issue by ReinhardKeil** [Open-CMSIS-Pack/devtools#828]
Relates to CMSIS-Toolbox 1.5 - Windows

`cbuild Hello.csolution.yml` generates a different structure in the directory ./out compared to the sequence:
```
csolution convert -s Hello.csolution.yml
cbuild Hello.Debug+AVH.cprj
```

`cbuild Hello.csolution.yml` does also not use the `output-dirs:` specification that can be added to the `*.csolution.yml` file (attached is the modified file).

[Hello.csolution.zip](https://github.com/Open-CMSIS-Pack/devtools/files/11209815/Hello.csolution.zip)

**Comment by soumeh01** [comment]
Thanks for pointing these issue.
**Issue1:** Regarding _different directory structure_ in` ./out` no more exists in the current version as it was already identified and fixed.

**Issue2:** cbuild doesn't use `output-dirs:` needs to be addressed.

**Comment by soumeh01** [comment]
Fixed in  [cmsis-toolbox 1.6.0](https://github.com/Open-CMSIS-Pack/cmsis-toolbox/releases/tag/1.6.0)
